src/EditorFeatures/CSharpTest/CodeRefactorings/UseExplicitOrImplicitType/UseExplicitTypeRefactoringTests.cs

```diff
@@ -423,6 +423,462 @@ static void Main()
             await TestInRegularAndScriptWhenDiagnosticNotAppliedAsync(code, expected);
         }
 
+        [Fact]
+        public async Task TestNullabilityAssignment1()
+        {
+            var code = @"
+#nullable enable
+
+class C
+{
+    private static string s_data;
+
+    static void Main()
+    {
+        var[||] v = s_data;
+    }
+}";
+
+            var expected = @"
+#nullable enable
+
+class C
+{
+    private static string s_data;
```

Comment:
Also have a test with no assignments other the initializer but with `string?` as the type? Just to validate that we're seeing the null assignment in that case?